doc/callbacks.md

```diff
@@ -8,53 +8,74 @@ as:
 
 Callbacks can be passed as an argument to the `Optimize()` function:
 
+<details>
+<summary>Click to check out the example</summary>
+<p>
```

Comment:
The `<p>` isn't necessary, I have the styling working with out.  So we can remove these. :)

Reply:
Sure